satellite/satellitedb/locked.go

```diff
@@ -845,10 +845,10 @@ func (m *lockedOverlayCache) UpdateStats(ctx context.Context, request *overlay.U
 }
 
 // UpdateUptime updates a single storagenode's uptime stats.
-func (m *lockedOverlayCache) UpdateUptime(ctx context.Context, nodeID storj.NodeID, isUp bool) (stats *overlay.NodeStats, err error) {
+func (m *lockedOverlayCache) UpdateUptime(ctx context.Context, nodeID storj.NodeID, isUp bool, alpha float64, beta float64, lambda float64, weight float64) (stats *overlay.NodeStats, err error) {
```

Comment:
For curiosity, how this file is updated and what it does?

Reply:
you can update this file by using `go generate` inside the `satellite/satellitedb` directory. I think it just puts a lock on the relevant database until the respective methods are finished, but I don't totally understand how it works.

Reply:
Ivan: There are issues with multi-threading in SQLite.  This file essentially generates a wrapper of the database code with a mutex to prevent those issues.  For more context, see this old, unfinished PR: https://github.com/storj/storj/pull/1513